graphstorm-processing/graphstorm_processing/data_transformations/dist_label_loader.py

```diff
@@ -73,19 +73,19 @@ class CustomSplit:
 
     Parameters
     ----------
-    train : str
-        Path of the training mask parquet file.
+    train : list[str]
```

Comment:
Why is train contains list[str], not all train/test/valid?

Reply:
Yeah need to fix to docstring.